libraries/SITL/SIM_Motor.cpp

```diff
@@ -212,9 +214,24 @@ float Motor::pwm_to_command(float pwm) const
 /*
   calculate thrust given a command value
 */
-float Motor::calc_thrust(float command, float air_density, float velocity_in, float voltage_scale) const
+float Motor::calc_thrust(float command, float air_density, float velocity_in, float voltage_scale)
 {
-    float velocity_out = voltage_scale * max_outflow_velocity * sqrtf((1-mot_expo)*command + mot_expo*sq(command));
+    /*
+      allow override of the expo
+     */
+    float expo = mot_expo;
+    const auto *sitl = AP::sitl();
+    if (sitl && !is_zero(sitl->vtol_motor_expo.get())) {
+        expo = sitl->vtol_motor_expo.get();
+    }
+
+    if (sitl && is_positive(sitl->vtol_motor_tc.get())) {
+        const float alpha = calc_lowpass_alpha_dt(1.0 / sitl->loop_rate_hz, 1.0 / sitl->vtol_motor_tc.get());
+        filtered_command += (command - filtered_command) * alpha;
+        command = filtered_command;
```

Comment:
We do already slew limit the command up on line 50. Maybe that could be setup to have the desired effect? In any case I think this filtering should be done in the same place as that. 

Also we do have the motor test example which allows "virtual" thrust testing, I was planning on using that to match up to real thrust data eventually.